modules/phase_field/src/userobjects/DiscreteNucleationMap.C

```diff
@@ -23,6 +23,7 @@ InputParameters validParams<DiscreteNucleationMap>()
 
 DiscreteNucleationMap::DiscreteNucleationMap(const InputParameters & parameters) :
     ElementUserObject(parameters),
+    _mesh_changed(false),
```

Comment:
Yeah, that sucks, but that was not the crash bug. Using ```i``` instead of ```qp``` to index a vector was. The missed initializations were fairly inconsequential (used in an if that only accessed initialized stuff either way and did the right thing either way).